### PR TITLE
fix(dnd) : #MAG-392 fix multiple things such as visuals and dnd on "My boards" folder, subsubfolders, ...

### PR DESCRIPTION
**Pull request by FlorentMr**
## Describe your changes
Fix multiple things such as visuals and dnd on "My boards" folder, subsubfolders, ...

## Checklist tests
Check ticket

## Issue ticket number and link
[MAG-392](https://jira.support-ent.fr/browse/MAG-392)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/folder-item/FolderItem.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const FolderItem: React.FunctionComponent<FolderListProps> = ({
   const [moveBoardsToFolder] = useMoveBoardsMutation();
   const [userRights] = useState<UserRights>(new UserRights(user));
   const [hasDrop, setHasDrop] = useState<boolean>(false);
-  const { folders, handleSelect, toggleSelect } = useFoldersNavigation();
+  const { folderData, handleSelect, toggleSelect } = useFoldersNavigation();
```
Comment (alicedraillard) [inline frontend/src/components/folder-item/FolderItem.tsx:39]: utilité d'avoir folderData ? on avait renommé en folders pour que ce soit + clair

File: frontend/src/components/tree-view/TreeViewContainer.tsx [hunk 2]
```diff
@@ -123,7 +123,7 @@ export const TreeViewContainer: React.FunctionComponent<
             title: MAIN_PAGE_TITLE,
             parentId: "",
           })
-        : folders.find(
+        : folderData.find(
```
Comment (alicedraillard) [inline frontend/src/components/tree-view/TreeViewContainer.tsx:126]: idem

**Review by Mattox0 — APPROVED** [review]

File: frontend/src/components/folder-item/FolderItem.tsx (hunk 1 above)
Comment (FlorentMr) [reply frontend/src/components/folder-item/FolderItem.tsx:39]: Justement, folders ne contient que les dossiers enfants du dossier actuel alors que folderData contient tous les dossiers, ce qui est nécessaire quand on cherche les infos du dossier dans lequel on se trouve (et qui est donc introuvable dans folders)

File: frontend/src/components/tree-view/TreeViewContainer.tsx (hunk 2 above)
Comment (FlorentMr) [reply frontend/src/components/tree-view/TreeViewContainer.tsx:126]: idem

**Review by alicedraillard — APPROVED** [review]

**Pull request merged**